scripts/github-export.py

```diff
@@ -0,0 +1,130 @@
+import json
+import csv
+import sys
+
+from typing import Optional
+
+from typer import Argument
+from typer import run
+
+import requests
```

Comment:
This package should be added to `pyproject.toml`:

    poetry add -G dev requests